src/Directory.Build.props

```diff
@@ -15,6 +15,7 @@
   <ItemDefinitionGroup Condition="'$(WinGetMacros)' != ''">
     <ClCompile>
       <PreprocessorDefinitions>$(WinGetMacros)%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <MultiProcessorCompilation>true</MultiProcessorCompilation>
```

Comment:
This should be in an unconditional `ItemDefinitionGroup`; this probably does nothing in the public repo since we don't set any of these values.